pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/ElasticSearchUtil.java

```diff
@@ -776,12 +776,23 @@ public static Map<String, Object> getObjectsMap(String index, String id, String
     }
 
     public static DDPParticipant getParticipantAsDDPParticipant(@NonNull Map<String, Map<String, Object>> participantsESData,
```

Comment:
Please rollback these changes! and check it in the `KitRequestShipping` class.
1) if the kit setting has care_of check the `ddpParticipant` if it has proxy guid
2) if proxy guid is not empty get the first-/lastname and add to ddpParticipant

Reply:
Honestly I think it makes more sense for it being here, we are getting the participant info from ES in this method and we know about the proxy guid here, in `KitRequestShipping` we only call this and get the ddpParticipant object that was already made

Reply:
Changes applied